Editor/Gui/Graph/GraphCanvas.cs

```diff
@@ -1038,29 +1047,29 @@ private void PasteClipboard()
                     }
                 }
             }
-            catch (Exception)
+            catch (Exception e)
             {
-                Log.Warning("Could not copy actual selection to clipboard.");
+                Log.Warning("Could not paste selection from clipboard.");
+                Log.Debug("Paste exception: " + e);
             }
         }
 
         private void DrawGrid()
         {
-            var color = new Color(0, 0, 0, 0.15f);
             var gridSize = Math.Abs(64.0f * Scale.X);
             for (var x = (-Scroll.X * Scale.X) % gridSize; x < WindowSize.X; x += gridSize)
             {
                 DrawList.AddLine(new Vector2(x, 0.0f) + WindowPos,
                                  new Vector2(x, WindowSize.Y) + WindowPos,
-                                 color);
+                                 GridColor);

```

Comment:
Stupid little thing that popped out at me when I was working on what's above.

a little more readable and the micro-est of optimizations

Reply:
valid point, I can un-cache. I just have difficulty seeing the risk, and see the potential for exposing this as a setting at a later date